tests/test_client_regress/tests.py

```diff
@@ -747,6 +764,14 @@ def test_nonform_error(self):
         for prefix, kwargs in self.msg_prefixes:
             self.assertFormsetError(self.response_nonform_errors, 'my_formset', None, None, msg, **kwargs)
 
+    def test_errors_none_deprecation(self):
+        """
+        A deprecation warning is emitted when using errors=None
+        """
+        deprecation_msg = "Passing errors=None to assertFormsetError() is deprecated. Use errors=[] instead."
+        with self.assertWarnsMessage(RemovedInDjango50Warning, deprecation_msg):
+            self.assertFormsetError(self.response_form_errors, 'my_formset', 0, 'value', None)
+
```

Comment:
I moved tests to `test_utils`.

Reply:
I left some suggestions about using `TestForm.invalid()` instead an an unbound `TestForm()`. Looks good otherwise, thanks 👍🏻